metadata-ingestion/source_docs/data_lake.md

```diff
@@ -0,0 +1,113 @@
+| `profiling.include_field_null_count`                 |                          | `True`       | Whether to profile for the number of nulls for each column.                                                                                                                                                                           |
+| `profiling.include_field_min_value`                  |                          | `True`       | Whether to profile for the min value of numeric columns.                                                                                                                                                                              |
+| `profiling.include_field_max_value`                  |                          | `True`       | Whether to profile for the max value of numeric columns.                                                                                                                                                                              |
+| `profiling.include_field_mean_value`                 |                          | `True`       | Whether to profile for the mean value of numeric columns.                                                                                                                                                                             |
+| `profiling.include_field_median_value`               |                          | `True`       | Whether to profile for the median value of numeric columns.                                                                                                                                                                           |
+| `profiling.include_field_stddev_value`               |                          | `True`       | Whether to profile for the standard deviation of numeric columns.                                                                                                                                                                     |
+| `profiling.include_field_quantiles`                  |                          | `True`       | Whether to profile for the quantiles of numeric columns.                                                                                                                                                                              |
+| `profiling.include_field_distinct_value_frequencies` |                          | `True`       | Whether to profile for distinct value frequencies.                                                                                                                                                                                    |
+| `profiling.include_field_histogram`                  |                          | `True`       | Whether to profile for the histogram for numeric fields.                                                                                                                                                                              |
```

Comment:
The defaults for some of these have changed in SQL profiling. Are they sufficiently cheap that we can still keep them enabled by default here?

Reply:
I ran some basic benchmarks on these – we should be able to enable the first two, but computing frequencies and histograms is much slower.